### PR TITLE
Sql foreign key checks enable and disable add to cheatsheet for SQL

### PR DESCRIPTION
**Pull request by parth93QA**

### CODE REVIEW

File: Linux_CheatSheet.md [hunk 1]
```diff
@@ -119,7 +119,18 @@ Linux/Unix:
 			#Options include
 				-n [NUM] = Print the last [NUM] lines instead of 10
 				-n +[NUM] = Print all lines starting with [NUM] until EOF
-		
+				
+       #Control System time and date
+                timedatectl [OPTIONS...] {COMMAND}
```
Comment (Jahenr) [inline Linux_CheatSheet.md:124]: Hi
I think this line can be removed.

**Review by Jahenr — CHANGES_REQUESTED** [review]
Please can you have a look at the comments, and make the changes when you can.

Thank you

**Comment by parth93QA** [comment]
Hi @Jahenr Have removed "-" from the beginnng of each timedatctl command. Please review once

**Comment by parth93QA** [comment]
Hi @Jahenr removed line 124

**Comment by Jahenr** [comment]
I have resolved the conflicts and merged it, thanks for the contribution.

**Pull request merged**